dandiapi/api/migrations/0022_applicationstats.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.2.23 on 2025-07-09 18:24
+from __future__ import annotations
+
+from django.db import migrations, models
+
+
+def populate_first_application_stats(apps, _):
```

Comment:
Since we don't want to ever have _nothing_ in the `ApplicationStats` table, I chose to create the first entry in the database migration, so that no matter what, a request can't sneak in and get an invalid response. Open to criticism, but this seemed like the most robust way to ensure this.

Reply:
I would suggest calling this `bootstrap_application_stats()` for a slightly clearer name.